test/e2e/runner/load.go

```diff
@@ -174,6 +188,43 @@ FOR_LOOP:
 	wg.Wait()
 }
 
+// loadProcess processes transactions by sending transactions received on the txCh
+// to the client.
+func loadProcessMultiple(ctx context.Context, txCh <-chan types.Tx, chSuccess chan<- struct{}, chFailed chan<- error, nodes []*e2e.Node, useInternalIP bool) {
```

Comment:
This new function could be easily be merged with `loadProcess`. That is, `loadProcess` is (almost) an instance of `loadProcessMultiple` when `nodes` has one element.

Reply:
Resolved with 14d0555f5